muikku-core-plugins/src/main/java/fi/otavanopisto/muikku/plugins/pedagogy/PedagogyController.java

```diff
@@ -213,5 +213,13 @@ public List<PedagogyFormHistory> listHistory(PedagogyForm form) {
     history.sort(Comparator.comparing(PedagogyFormHistory::getCreated).reversed());
     return history;
   }
+  
+  public Boolean getHasPedagogyForm(String studentIdentifier) {
```

Comment:
I'd still like to see this optimized as...

```
public boolean hasPedagogyForm(String studentIdentifier) {
  return findFormByStudentIdentifier(studentIdentifier) != null;
}
```

so Java doesn't have to create variables that are immediately garbage collected after leaving this method.